### ISSUE TITLE
Question: Can Powermail catch errors from the mail system?

### ISSUE THREAD

**Issue by onkelbeh** [einpraegsam/powermail#749]
Hi,

hope you don't mind questions here:
Powermail seems to ignore the delivery status of a generated Email.

Two cases tested:
1) Typo3 Email configuration set to "smtp", using a postfix on localhost as SMTP target, if I just kill the local postfix, Typo3 reports an error while trying to send an Email.
2) Email configured with "sendmail", for the test, I replaced the local /usr/sbin/sendmail with a script which always exits with an error. Typo3 reports an error while trying to send an Email.

My problem:

In both cases, Powermail _always_ reports a **successfully** sent Email on the frontend, though Typo3 seems to recognise the problem with the mail system. Am I missing something? Is there a way for Powermail to report a failed mail delivery to the user at the frontend?

As a workaround, we could use the database from the backend, but would require that "recipients" have access to the backend, sending a mail would be much easier.

Thanks a lot in advance.

**Comment by einpraegsam** [comment]
Please don't ask support questions in github. Pls use Slack or Stackoverflow for this. Otherwise the issue list is even more flooded with tasks for us.

**Comment by glucka** [comment]
I think this is really a serious problem! Sometimes after weeks realize you there is a problem.

**Comment by onkelbeh** [comment]
Yep, after loosing a bunch of customer mails. The backend's exit code simply is ignored by powermail, but this does not matter if you are working in a marketing department.
